docker-compose.yml

```diff
@@ -38,12 +39,14 @@ services:
       PS_FOLDER_INSTALL: ${PS_FOLDER_INSTALL:-install-dev}
       PS_FOLDER_ADMIN: ${PS_FOLDER_ADMIN:-admin-dev}
       PS_COUNTRY: ${PS_COUNTRY:-fr}
-      PS_LANGUAGE: ${PS_LANGUAGE:-fr}
-      PS_DEV_MODE: ${PS_DEV_MODE:-0}
+      PS_LANGUAGE: ${PS_LANGUAGE:-en}
+      PS_DEV_MODE: ${PS_DEV_MODE:-1}
```

Comment:
No need for this change IMO. It will be customizable in .env file after #31527 is merged

Reply:
lets see which one gets merged first :D

Reply:
Matters not which is first, but what will it be in the end. Although english is more common than french and dev mode  should be enabled by default for development so this change is still an improvement.